src/core/router.go

```diff
@@ -218,6 +220,9 @@ func initRouters() {
 	beego.Router("/api/scans/all/metrics", scanAllAPI, "get:GetScanAllMetrics")
 	beego.Router("/api/scans/schedule/metrics", scanAllAPI, "get:GetScheduleMetrics")
 
+	// Add handler for api v2.0
+	beego.Handler("/api/v2.0/*", handler.New())
```

Comment:
There is bug for beego when binding the `http.Handler` directly: https://github.com/astaxie/beego/issues/3906

I wrap the `http.Handler` to `beego.Controller` in PR https://github.com/goharbor/harbor/pull/10440, just for refererence

Reply:
There maybe some easier solution for the bug. I opened an issue to track. Let's fix it in another PR